api/src/pcapi/alembic/versions/20230522T100913_dcfd7657a244_add_educational_redactor_column_to_collective_offer_request.py

```diff
@@ -0,0 +1,30 @@
+"""Add-educationalRedactorId to collective_offer_request table
```

Comment:
il reste un tiret inutile

```suggestion
"""Add educationalRedactorId to collective_offer_request table
```